commit efa498c13b883810497e0ea8a169efd6f48f5026
Author: Jia Tan <jiat0218@gmail.com>
Date:   Fri Feb 17 00:36:05 2023 +0800

    liblzma: Rename field => member in documentation.
    
    Also adjusted preset value => preset level.

diff --git a/src/liblzma/api/lzma/base.h b/src/liblzma/api/lzma/base.h
index 9e513450..62d86d84 100644
--- a/src/liblzma/api/lzma/base.h
+++ b/src/liblzma/api/lzma/base.h
@@ -548,16 +548,16 @@ typedef struct {
 	 * may change.
 	 */
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr4;
 
 	/**
@@ -572,19 +572,19 @@ typedef struct {
 	 */
 	uint64_t seek_pos;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint64_t reserved_int2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	size_t reserved_int3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	size_t reserved_int4;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum2;
 
 } lzma_stream;
diff --git a/src/liblzma/api/lzma/block.h b/src/liblzma/api/lzma/block.h
index fc0a4508..7080301f 100644
--- a/src/liblzma/api/lzma/block.h
+++ b/src/liblzma/api/lzma/block.h
@@ -238,49 +238,49 @@ typedef struct {
 	 * uninitialized.
 	 */
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_vli reserved_int3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_vli reserved_int4;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_vli reserved_int5;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_vli reserved_int6;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_vli reserved_int7;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_vli reserved_int8;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum4;
 
 	/**
@@ -304,25 +304,25 @@ typedef struct {
 	 */
 	lzma_bool ignore_check;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool4;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool5;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool6;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool7;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool8;
 
 } lzma_block;
diff --git a/src/liblzma/api/lzma/container.h b/src/liblzma/api/lzma/container.h
index fcbe5734..cbd07bb2 100644
--- a/src/liblzma/api/lzma/container.h
+++ b/src/liblzma/api/lzma/container.h
@@ -172,25 +172,25 @@ typedef struct {
 	 * with the currently supported options, so it is safe to leave these
 	 * uninitialized.
 	 */
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int4;
 
 	/**
@@ -236,22 +236,22 @@ typedef struct {
 	 */
 	uint64_t memlimit_stop;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint64_t reserved_int7;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint64_t reserved_int8;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr4;
 
 } lzma_mt;
diff --git a/src/liblzma/api/lzma/delta.h b/src/liblzma/api/lzma/delta.h
index 8b8ae5c6..ad1aca90 100644
--- a/src/liblzma/api/lzma/delta.h
+++ b/src/liblzma/api/lzma/delta.h
@@ -77,22 +77,22 @@ typedef struct {
 	 * uninitialized.
 	 */
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int4;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr2;
 
 } lzma_options_delta;
diff --git a/src/liblzma/api/lzma/index.h b/src/liblzma/api/lzma/index.h
index 64ec8350..73bad8c1 100644
--- a/src/liblzma/api/lzma/index.h
+++ b/src/liblzma/api/lzma/index.h
@@ -51,13 +51,13 @@ typedef struct {
 		 */
 		const lzma_stream_flags *flags;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		const void *reserved_ptr1;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		const void *reserved_ptr2;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		const void *reserved_ptr3;
 
 		/**
@@ -114,16 +114,16 @@ typedef struct {
 		lzma_vli padding;
 
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		lzma_vli reserved_vli1;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		lzma_vli reserved_vli2;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		lzma_vli reserved_vli3;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		lzma_vli reserved_vli4;
 	} stream;
 
@@ -210,28 +210,28 @@ typedef struct {
 		 */
 		lzma_vli total_size;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		lzma_vli reserved_vli1;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		lzma_vli reserved_vli2;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		lzma_vli reserved_vli3;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		lzma_vli reserved_vli4;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		const void *reserved_ptr1;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		const void *reserved_ptr2;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		const void *reserved_ptr3;
 
-		/** \private     Reserved field. */
+		/** \private     Reserved member. */
 		const void *reserved_ptr4;
 	} block;
 
diff --git a/src/liblzma/api/lzma/lzma12.h b/src/liblzma/api/lzma/lzma12.h
index dcc762df..3d3723f9 100644
--- a/src/liblzma/api/lzma/lzma12.h
+++ b/src/liblzma/api/lzma/lzma12.h
@@ -492,37 +492,37 @@ typedef struct {
 	 * uninitialized.
 	 */
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int4;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int5;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int6;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int7;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int8;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum4;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	void *reserved_ptr2;
 
 } lzma_options_lzma;
@@ -550,13 +550,13 @@ do { \
  * The flags are defined in container.h, because the flags are used also
  * with lzma_easy_encoder().
  *
- * The preset values are subject to changes between liblzma versions.
+ * The preset levels are subject to changes between liblzma versions.
  *
  * This function is available only if LZMA1 or LZMA2 encoder has been enabled
  * when building liblzma.
  *
- * \param[out]  options Pointer to LZMA12 options to be filled
- * \param       preset  Preset value between 0-9 ORed with preset flags
+ * \param[out]  options Pointer to LZMA1 or LZMA2 options to be filled
+ * \param       preset  Preset level bitwse-ORed with preset flags
  *
  * \return      lzma_bool:
  *              - true if the preset is not supported (failure).
diff --git a/src/liblzma/api/lzma/stream_flags.h b/src/liblzma/api/lzma/stream_flags.h
index c4923dd6..e259b085 100644
--- a/src/liblzma/api/lzma/stream_flags.h
+++ b/src/liblzma/api/lzma/stream_flags.h
@@ -70,12 +70,12 @@ typedef struct {
 	lzma_vli backward_size;
 
 	/**
-	 * \brief       Minimum size of the backward_size field
+	 * \brief       Minimum size of the backward_size member
 	 */
 #	define LZMA_BACKWARD_SIZE_MIN 4
 
 	/**
-	 * \brief       Maximum size of the backward_size field
+	 * \brief       Maximum size of the backward_size member
 	 */
 #	define LZMA_BACKWARD_SIZE_MAX (LZMA_VLI_C(1) << 34)
 
@@ -97,46 +97,46 @@ typedef struct {
 	 * nice to have the proper types when they are needed.)
 	 */
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_reserved_enum reserved_enum4;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool2;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool3;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool4;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool5;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool6;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool7;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	lzma_bool reserved_bool8;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int1;
 
-	/** \private     Reserved field. */
+	/** \private     Reserved member. */
 	uint32_t reserved_int2;
 
 } lzma_stream_flags;